desktop/apps/artwork/components/current_auctions/current_auctions.jsx

```diff
@@ -0,0 +1,83 @@
+import React, { PropTypes } from 'react'
+import { partition, take } from 'underscore'
+import { upcomingLabel } from '../../lib/auction_label'
+
+export default function CurrentAuctions ({ auctionContextId, sales }) {
+  const sortedSales = sortSales(sales, auctionContextId)
+
+  return (
+    <section className='artwork-current-auctions'>
+      <div className='artwork-current-auctions__header-area'>
+        <h3>
+          Current Auctions on Artsy
+        </h3>
+
+        <a href='/auctions'>
+          View All Auctions
+        </a>
+      </div>
+
+      <div className='artwork-current-auctions__sales'>
+        {sortedSales.map((sale) => {
+          const {
+            cover_image: {
+              cropped: {
+                url
+              }
+            },
+            end_at,
+            href,
+            id,
+            name,
+            live_start_at,
+            start_at,
+            status
+          } = sale
+
+          const statusLabel = 'Auction ' + upcomingLabel(start_at, end_at,
+                                                         live_start_at, status)
+          return (
+            <div className='artwork-current-auctions__sale-item' key={id}>
+              <a href={href}>
+                <img src={url} alt={name} />
+
+                <div className='artwork-current-auctions__label-container'>
+                  <div className='artwork-current-auctions__name'>
+                    {name}
+                  </div>
+
+                  <div className='artwork-current-auctions__status'>
+                    {statusLabel}
+                  </div>
+                </div>
+              </a>
+            </div>
+          )
+        })}
+      </div>
+    </section>
+  )
+}
+
+CurrentAuctions.propTypes = {
```

Comment:
Very strongly in favor of `propTypes` -- they're the contract that binds your component to its use in the outside world, as well as provides a lot of self-documenting properties. If you tag on `.isRequired` and fail to pass in a prop, an error will throw in the console -- how could that be wrong? :)